packages/frontend-shared/src/gql-components/CopyButton.vue

```diff
@@ -26,11 +25,11 @@
 </template>
 
 <script setup lang="ts">
-import { useClipboard } from '@vueuse/core'
+import { useClipboard } from './useClipboard'
```

Comment:
The problem is in us and firefox not in vueuse:
- For security reasons, Firefox  prevents the use of the clipboard if you are not in https
- Chrome prevents the use of the clipboard if you are not focused on the current window and the click you make is not "real"

These issue make the "isSupported" flag false and make the button disapear in most situations

Reply:
> Locally-delivered resources such as those with http://127.0.0.1 URLs, http://localhost and http://*.localhost URLs (e.g. http://dev.whatever.localhost/), and file:// URLs are also considered to have been delivered securely.

https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts#when_is_a_context_considered_secure

> disabled From version 87: this feature is behind thedom.events.asyncClipboard.clipboardItem preferences (needs to be set to true). To change preferences in Firefox, visit about:config.
> footnote Writing to the clipboard is available without permission in secure contexts and browser extensions, but only from user-initiated event callbacks. Browser extensions with the "clipboardWrite" permission can write to the clipboard at any time.

https://developer.mozilla.org/en-US/docs/Web/API/Clipboard

So it looks like it might theoretically be possible to get it working on Firefox, but moving it to the server works in all cases and may be simpler because of that. Gotcha.

Reply:
I think I remember now that their isSupported check doesn't properly recognizing that localhost is considered a secure context - it was a couple months ago that I looked into it.